docs/iris/src/whatsnew/index.rst

```diff
@@ -7,10 +7,8 @@ These "What's new" pages describe the important changes between major
 Iris versions.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
-   latest.rst
-   3.0.rst
```

Comment:
@tkknight We need to reinstate these...

Reply:
This has been reinstated.  However this does introduce a couple warnings into the build:

```
whatsnew/index.rst:9: WARNING: toctree contains reference to nonexisting document 'whatsnew/latest'
index.rst:9: WARNING: toctree contains reference to nonexisting document 'whatsnew/3.0'
```

This is because the **index.rst** is referencing files that are not there unless the **whatsnew/aggregate_directory.py** is run to create them.  

This is ok for now.  A longer term change can be applied another time to address this, then we will have a **warning free** build - yay!

Reply:
> A longer term change can be applied another time to address this

Can I suggest we raise an issue for the longer term change?

Reply:
I plan to - I have a few potential changes that I need to think through.  this is high on the list and I will create an issue for it soon.